docs/diagrams/join.md

```diff
@@ -0,0 +1,41 @@
+title etcdadm join https://member1:2379
+
+participant user
+participant remote repo
+participant etcdadm
+participant systemd
+database member1
+database member2
```

Comment:
Because the sequence diagram is showing the interaction between multiple processes on one host (etcdadm, systemd, member2) and also between hosts, it feels a bit confusing. It looks like the website you're using has a `participantgroup` concept. Do you think grouping the process participants by host would help?

Reply:
Yes! Great idea. Will add this.